scripts/code_size_compare.py

```diff
@@ -208,23 +212,23 @@ def _gen_code_size_report(self, revision, git_worktree_path):
                 ["size -t library/libmbedcrypto.a"], cwd=git_worktree_path, shell=True
             )
         except subprocess.CalledProcessError as e:
-            self._handle_CalledProcessError(e,git_worktree_path)
+            self._handle_called_process_error(e, git_worktree_path)
         crypto_text = result.decode()
         # Size for libmbedx509.a
         try:
             result = subprocess.check_output(
                 ["size -t library/libmbedx509.a"], cwd=git_worktree_path, shell=True
             )
         except subprocess.CalledProcessError as e:
-            self._handle_CalledProcessError(e,git_worktree_path)
+            self._handle_called_process_error(e, git_worktree_path)
         x509_text = result.decode()
         # Size for libmbedtls.a
         try:
             result = subprocess.check_output(
                 ["size -t library/libmbedtls.a"], cwd=git_worktree_path, shell=True
             )
         except subprocess.CalledProcessError as e:
-            self._handle_CalledProcessError(e,git_worktree_path)
+            self._handle_called_process_error(e, git_worktree_path)
```

Comment:
Fixing pylint should ideally be done in a separate commit. This commit was primarily about removing f-strings so really it should only do this one thing.

As is the case with my comment around the f-strings, it would be better to fix the pylint related errors in their original commit, via an interactive rebase, rather than applying a fix-up commit to the end since this will generate a cleaner git history.

Reply:
Yes, ideally that would be in a separate commit, but at this point, let's not use Aditya's limited time rewriting the commit history